parts/windows/csecmd.ps1

```diff
@@ -16,7 +16,7 @@ $arguments = '
 -CSEResultFilePath %SYSTEMDRIVE%\AzureData\CSEResult.log';
 $inputFile = '%SYSTEMDRIVE%\AzureData\CustomData.bin';
 $outputFile = '%SYSTEMDRIVE%\AzureData\CustomDataSetupScript.ps1';
-if (!(Test-Path $inputFile)) { echo 49 | Out-File -FilePath '%SYSTEMDRIVE%\AzureData\CSEResult.log' -Encoding utf8; exit; };
+if (!(Test-Path $inputFile)) { throw 'ExitCode: `"49`", Output: `"WINDOWS_CSE_ERROR_NO_CUSTOM_DATA_BIN`", Error: `"%SYSTEMDRIVE%\AzureData\CustomData.bin does not exist.`"' };
 Copy-Item $inputFile $outputFile;
 Invoke-Expression('{0} {1}' -f $outputFile, $arguments);
-\" >> %SYSTEMDRIVE%\AzureData\CustomDataSetupScript.log 2>&1; if (!(Test-Path %SYSTEMDRIVE%\AzureData\CSEResult.log)) { exit 50; }; $code=(Get-Content %SYSTEMDRIVE%\AzureData\CSEResult.log); exit $code
\ No newline at end of file
+\"
```

Comment:
If we change `exit $code` to `throw $code`, 2>&1 will redirect the exception to the log file `%SYSTEMDRIVE%\AzureData\CustomDataSetupScript.log` and exit 0. So we have to use `Start-Transcript -Path $LogFile` for logging.